pkg/node/helper.go

```diff
@@ -555,3 +560,75 @@ func GetNodeData(endpoint string) (
 		"0x" + hex.EncodeToString(proofOfPossession.ProofOfPossession[:]),
 		nil
 }
+
+func SeedClusterData(
```

Comment:
Can it have another name. Something related to downloading blockchain db for the cluster

Reply:
changed to DownloadPublicArchive